provd/proto/eula.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package eula;
+
+// Just a placeholder until we have designed the EULA
+service EULAService {
+    rpc AcceptEULA(AcceptEULARequest) returns (Empty) {}
+}
+
+message Empty {}
+
+message AcceptEULARequest {
+    string eulaId = 1;
+    string username = 2;
```

Comment:
Is there a reason behind providing the username in the response and will there be more than one Eula on the machine which means we need to have an Id.

Perhaps a question for the OEM team; have you spoken with them?

Is convention here `AcceptEulaRequest` or `AccpetEULAReqest?`

Reply:
Getting final clarification from the OEM team, but for now it doesn't look like we need to actually pass anything in